src/uk/gov/hmcts/contino/Builder.groovy

```diff
@@ -9,6 +9,7 @@ interface Builder {
   def performanceTest()
   def apiGatewayTest()
   def crossBrowserTest()
+  def crossBrowserTest(String browser)
```

Comment:
cant this be the only method and call it with empty string to keep the behaviour of   `def crossBrowserTest()` intact?

Reply:
good call - will change this